### PR TITLE
Some corrections and fixes of the staging best practices

### PR DESCRIPTION
**Pull request by krauselukas**

### CODE REVIEW

File: xml/obs_best_practice_staging_workflow.xml [hunk 1]
```diff
@@ -36,8 +36,8 @@
   <sect2>
     <title>Start using it</title>
     <para>
-      In this view, we could find all the staging projects with a request
-      associated and also if the requests have some problems.
+      In this view, we can find all the staging projects with a request
+      associated and the current state of them.
```
Comment (dmarcoux) [inline xml/obs_best_practice_staging_workflow.xml]: with an associated request and their current state.

Comment (krauselukas) [reply xml/obs_best_practice_staging_workflow.xml]: @dmarcoux changed it :)

File: xml/obs_best_practice_staging_workflow.xml [hunk 2]
```diff
@@ -122,7 +122,7 @@
       <listitem>
         <para>
           <emphasis>Obsolete Requests:</emphasis>
-          Requests that was declined, revoked or superseded.
+          Requests that were declined, revoked or superseded.
```
Comment (krauselukas) [inline xml/obs_best_practice_staging_workflow.xml]: @dmarcoux not 100% sure, but i think it need to be "were"

Comment (dmarcoux) [reply xml/obs_best_practice_staging_workflow.xml]: Yes

**Pull request merged**